README.md

```diff
@@ -1,3 +1,141 @@
 # blinkmobile / angular-location [![npm](https://img.shields.io/npm/v/@blinkmobile/angular-location.svg?maxAge=2592000)](https://www.npmjs.com/package/@blinkmobile/angular-location) [![Travis CI Status](https://travis-ci.org/blinkmobile/angular-location.js.svg?branch=master)](https://travis-ci.org/blinkmobile/angular-location.js)
 
 Angular 1.x components for maps and geolocation
+
+
+## API
+
+
+### Configuration
+
+You must define a "googleMapsApiKey" property on the `$rootScope` with your API key.
```

Comment:
yeah i agree with handiness of having it on rootscope for multiple controls, but I also agree with wanting to follow the conventions and good practices. The primitive trap is something most angular developers should be familiar with. 

so i'll take the easy way out here and say the choice is yours, i dont think its a deal breaker